### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/nuxt#18525] [Announcements]
> **3.1.1** is a bugfix release to address a problem rendering components injected by Vue or Nuxt plugins.

## Update notes

There's also a [Nitro upgrade to v2.1.0](https://github.com/unjs/nitro/blob/main/CHANGELOG.md) released shortly after v3.1.1, so when upgrading, please either run `nuxt upgrade --force` or refresh your lockfile.

## Changelog

[compare changes](https://github.com/nuxt/nuxt/compare/v3.1.0...v3.1.1)

**Comment by lkul2912** [comment]
Chào bạn. Cám ơn bạn đã trao cơ hội để tôi có thể tâm sự giao lưu nghiên cứu cùng bạn. Nó có thể đi xa hơn về khoa học và thực tiển tốt rất nhiều so với việc đơn thuần chỉ áp dụng vào nguồn thuỷ hải sản. Tôi chấp nhận nghiên cứu của bạn sẽ nếu có thời gian hay cùng tôi trao đổi để tôi quyết định cho nghiên cứu đó có hoàn thành tuyệt đối và có đưa vào thật tế được chưa thì. Thật tế đã OK nhưng tôi muốn chúng ta cùng nhau làm tốt hơn cho nó để ứng dụng cào truyền thống và tạo tác cho những lễ hội sau này tại mọi nơi.
